.github/workflows/tests.yml

```diff
@@ -323,17 +323,22 @@ jobs:
     if: ${{ !failure() && !cancelled() }}
     needs: linting-done
     runs-on: ubuntu-latest
-    container:
-      # https://github.com/matrix-org/complement/blob/master/dockerfiles/ComplementCIBuildkite.Dockerfile
-      image: matrixdotorg/complement:latest
-      env:
-        CI: true
-      ports:
-        - 8448:8448
-      volumes:
-        - /var/run/docker.sock:/var/run/docker.sock
 
     steps:
+      # Env vars are set file a file given by $GITHUB_PATH. We need both Go 1.17 and GOPATH on env to run Complement.
```

Comment:
> Env vars are set file a file given by $GITHUB_PATH

Sorry I'm having troubles parsing this sentence, could you clarify it a bit please?

Reply:
not just any env var - specifically `PATH`, aiui.

Reply:
Clarified